CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ endmacro()
 GroupSources(${PROJECT_SOURCE_DIR}/src "Source Files")
 
 include_directories(
-  ${ALTV_JS_CPP_SDK}
```

Comment:
This variable should remain to be used as it is possible to specify a custom path to cpp sdk